ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/InternalCustomerCenter.kt

```diff
@@ -75,12 +75,31 @@ private fun CustomerCenterError(state: CustomerCenterState.Error) {
     Text("Error: ${state.error}")
 }
 
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
 @Composable
 private fun CustomerCenterLoaded(state: CustomerCenterState.Success) {
-    // CustomerCenter WIP: Add proper success UI
-    Column {
-        Text("Customer Center config:", fontWeight = FontWeight.Bold, fontSize = 20.sp)
-        Text(state.customerCenterConfigDataString)
+    val configuration = state.customerCenterConfigData
+
+    if (state.purchaseInformation != null) {
+        if (configuration.screens.containsKey(CustomerCenterConfigData.Screen.ScreenType.MANAGEMENT)) {
```

Comment:
I'm wondering if it should be a parsing error for the json not to contain this and/or the NO_ACTIVE screens instead of having to force unwrap here...

In that case, we should maybe make accesors in the `CustomerCenterConfigData` or convert `screens` to a different type that has ready accessors... 

All of this can be done in different PRs if we want to however, so not a blocker for this PR.

Reply:
I think an error makes sense tbh. If we don't get those screens it's clearly an error

Reply:
Right, I'm just wondering if we should fail early, at the parsing the response phase... But we can keep this for now.